src/modules/RegisterFormModule/index.tsx

```
@@ -61,70 +61,86 @@ export const RegisterModule = () => {
+    if (currentStepConfig?.validate && currentStepConfig.keys) {
+      const stepDataToValidate: Partial<RegisterFormData> = {};
+      currentStepConfig.keys.forEach((key) => {
+        const value = formData[key];
+        if (
+          key === 'ktpFile' ||
+          key === 'npwpFile' ||
+          key === 'diplomaFile' ||
+          key === 'skkFile' ||
+          key === 'profilePhoto'
+        ) {
+          stepDataToValidate[key] = value === null ? undefined : (value as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)